data/mods/Aftershock/items/ammo/25mm.json

```diff
@@ -19,5 +19,40 @@
     "dispersion": 75,
     "recoil": 2200,
     "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "afs_25mm_frag",
+    "type": "AMMO",
+    "copy-from": "afs_25mm_shot",
+    "name": { "str_sp": "25mm FRAG" },
+    "description": "Standard issue UICA 25mm rounds, each loading a plastic explosive charge within a frangible ceramic projectile.",
+    "//": "expensive illegal ammo",
+    "price": "1750 USD",
```

Comment:
If I'm reading this correctly the rounds are more expensive than the gun?

Reply:
Yeah I imagined these were military only, so any you manage to buy is basically blackmarket ammo. They might need a better price tho, ill check some irl references if I find them.

Reply:
So turns out a 40mm grenade is worth about $200-$300 in the market. Figured that around x5 times that price was a reasonable blackmarket rate, and adjusted the price to that.